submissions/YanaGushlevskaya/js-DOM/src/assets/scss/_mixins.scss

```diff
@@ -0,0 +1,49 @@
+$breakpoint-small: 576px;
+$breakpoint-medium: 768px;
+$breakpoint-large: 992px;
+$breakpoint-extra: 1200px;
```

Comment:
CSS breakpoints is good,
[Here](https://www.freecodecamp.org/news/the-100-correct-way-to-do-css-breakpoints-88d6a5ba1862/) is a usefull link, maybe you will find smth for you.

Reply:
Thanks, it is interesting about "up" boundaries and "right" naming of varibles for resolution mixins